pages/about.html

```diff
@@ -0,0 +1,155 @@
+        <div class="menu-icon-container">
+          <img
+            src="../images/icons/mobile_menu_button.png"
+            alt="menu-icon"
+            id="menu-icon"
+          />
+        </div>
+        <div class="hero-title">
+          <h2>"Hello! Sharing world"</h2>
+          <h1>CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+        </div>
+        <div class="message">
+          <p>
+            The Global Summit brings together a community of experts, academics,
+            and activists who comprise the Creative Commons affiliate network in
+            a different country every year. This year, we hope to expand our
+            invitation list including organizations and individuals.
+          </p>
+        </div>
+        <div class="contact">
+          <p>
+            Please contact us per Email for any further questions about CC
+            Global Summit 2015!
+          </p>
+          <h4>ccsummit2015cckorea.org</h4>
+        </div>
+      </section>
+      <section id="logo-creation">
+        <div class="logo-creation-title">
+          <h5>Creative Commons Global Summit 2015 Logo</h5>
+          <span></span>
+        </div>
+        <p>
+          The logo of CC Global Summit 2015 was decided through the logo
+          competition from 8 June to 7 July.
+        </p>
+        <div class="summit-logo">
+          <img
+            src="../images/about-page/about_logo-summit_logo.png"
+            alt="summit-logo"
+          />
+        </div>
+      </section>
+      <hr />
+      <section class="previous-global-summits">
+        <div class="previous-global-summits-title">
+          <h5>See the past CC Global Summits</h5>
+          <span></span>
+        </div>
+        <p>
+          Take a look at the last two CC Global Summits which took place in
+          Buenos Aires and in Warsaw
+        </p>
+        <div class="previous-global-summits-image-container">
+          <img src="../images/about-page/about_2013.png" alt="summit-2013" />
+          <img src="../images/about-page/about_2011.png" alt="summit-2011" />
+        </div>
+      </section>
+      <section id="partners-section">
+        <div class="partners-title">
+          <h5>Partners</h5>
+          <span></span>
+        </div>
+        <div class="images-container">
+          <img src="../images/partners/google-logo.png" alt="google-logo" />
+          <img src="../images/partners/airbnb-logo.png" alt="airbnb" />
+          <img src="../images/partners/mozilla.png" alt="mozilla" />
+          <img src="../images/partners/naver.png" alt="naver" />
+          <img src="../images/partners/wikihow.png" alt="wikihow" />
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-image">
+        <img src="../images/icons/cck-logo1.png" alt="footer-image" />
+      </div>
+      <div class="footer-text">
+        <p>2015 Creative Commons Korea.</p>
+        <p>Some Rights Reserved.</p>
+      </div>
+    </footer>
```

Comment:
- [ ] According to the design template, the footer section on the desktop view has a dark background color and white text and logo, and there is no partners section. Kindly change the style of your project so it can look similar to the template you are following(the partners' section should be preserved for the mobile view and the background color of the footer on the mobile view is white and it is correct in your project).